### PR TITLE
Issue 33297: Eliminate AspectMock from DataExtensionUtilTest

### PR DESCRIPTION
**Pull request by karyna-tsymbal-atwix**
### Description
Eliminated AspectMock usage from `dev/tests/unit/Magento/FunctionalTestFramework/DataGenerator/Util/DataExtensionUtilTest.php`

### Fixed Issues (if relevant)
1. Fixes magento/magento2#33297: [MFTF] Eliminate AspectMock from DataExtensionUtilTest (Complex!)

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

**Review by bohdan-harniuk — APPROVED** [review]
Hello, @jilu1

This task was finished without creating new singletons, so it is not a blocker. You can proceed with the code review.

Thanks, Bohdan

File: dev/tests/unit/Util/MagentoTestCase.php [hunk 1]
```diff
@@ -19,6 +19,8 @@ public static function setUpBeforeClass(): void
         if (!self::fileExists(DOCS_OUTPUT_DIR)) {
             mkdir(DOCS_OUTPUT_DIR, 0755, true);
         }
+        // Should be used to clean AspectMock mocking before using PHPUnit mocking and Reflection.
+        AspectMock::clean();
```
Comment (jilu1) [inline dev/tests/unit/Util/MagentoTestCase.php:23]: This Aspect Mock usage needs to be removed.

**Review by jilu1 — CHANGES_REQUESTED** [review]
Any reason you closed PR #https://github.com/magento/magento2-functional-testing-framework/pull/849?

**Comment by jilu1** [comment]
Change looks good. Please address the comment inlined.

File: dev/tests/unit/Util/MagentoTestCase.php (hunk 1 above)
Comment (bohdan-harniuk) [reply dev/tests/unit/Util/MagentoTestCase.php:23]: Hello, @jilu1 
It is temporarily. When all PRs merged we will proceed with the AspectMock eliminating in this class.

Comment (jilu1) [reply dev/tests/unit/Util/MagentoTestCase.php:23]: @bohdan-harniuk 
Got it. Thanks for the explanation!

**Review by jilu1 — APPROVED** [review]

**Comment by jilu1** [comment]
@magento import pull request to https://github.com/magento-commerce/magento2-functional-testing-framework

**Comment by magento-engcom-team** [comment]
@jilu1 the pull request successfully imported.

**Pull request merged**